docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+
+:::tip
+This variables applies only to your current shell session. If you want the variable to apply to future shell sessions, set the variable in your shell startup file, for example in the `~/.bashrc` or `~/.profile` file.
+:::
+
+:::note
+The following steps assumes you have completed the [Install Nebari](/started/installing-nebari) section, has confirmed that `nebari` is successfully installed in your environment and opted for **AWS** as your cloud provider. as well as has already configured the `nebari` environment variables. If you had any issues during the installation, please visit our getting started [troubleshooting](/started/troubleshooting) section for extra advice.
+:::
+
+
+## Nebari Initialize
+
+Great, you’ve gone through the [Nebari Installation](/started/installing-nebari.md) and [Authentication setup](#authentication) steps, and have ensured that all the necessary environment variables have been properly set. It is time to initialize and deploy Nebari!
+
+In your terminal, we advise you to start by creating a new project folder. Here, we will name the new folder as `nebari-aws`:
+
+```bash
+mkdir nebari-aws && cd nebari-aws
+```
+When you first initialize Nebari, you will be creating a `nebari-config.yaml` that contains a collection of preferences and settings for your deployment. The command bellow will generate a basic config file with an infrastructure based on **AWS**, named **projectname**, where the domain will be **domain** and the authetication mode set to **password**.
+
+```bash
+nebari init aws --project projectname --domain domain --auth-provider password
+```
+:::note Note
+You will be prompted to enter values for some of the choices above if they are omitted as command line arguments (for example project name and domain)
+:::
+
+Once `nebari init` is executed, you should then be able to see the following output:
+```bash
+Securely generated default random password=*** for Keycloak root user
+stored at path=/tmp/NEBARI_DEFAULT_PASSWORD
+INFO:botocore.credentials:Found credentials in environment variables.
+```
+:::tip
+MacOS generates a programmatic directory stored in `/private/var` and defines the `$TMPDIR` environment variable for locating the system temporary folder. We advise the user to look for the `NEBARI_DEFAULT_PASSWORD` file in the following  `/var/folders/xx/xxxxx/T` path.
+:::
+
+You can see that Nebari is generating a random password for the root user of Keycloak. This password is stored in a temporary file and will be used to authenticate to the Keycloak server once Nebari's infrastructure is fully deployed.
+
+The generated `nebari-config.yaml` is the configuration file that will determine how the cloud infrastructure and Nebari is built and deployed in the next step. But at this point it’s just a text file. You could edit it manually if you are unhappy with the choices, or delete it and start over again.
```

Comment:
```suggestion
The generated `nebari-config.yaml` is the configuration file that will determine how the cloud infrastructure and Nebari is built and deployed in the next step. Since it is a simple text file, you can edit it manually if you are unhappy with the choices you made during initialization, or delete it and start over again by re-running `nebari init`.
```